.circleci/config.yml

```diff
@@ -74,6 +74,14 @@ commands:
       - run:
           name: Unit test
           command: bash -l -c "make unit_test"
+          # temporary - to remove before merging the PR
+      - run:
+          name: Download pre-generated index
+          command: wget -q -P tests/benchmark/data https://s3.amazonaws.com/redismodules/VectorSimilarity/random-1M-100-l2.hnsw
+      - run:
+           name: Benchmark
+           command: bash -l -c "make benchmark"
+           no_output_timeout: 180m
```

Comment:
need to remove this?

Reply:
of course, this is here until we finish the review, to see that it works in CI before we merge it...